basics-assignments-4-start 2/src/app/my-outer/my-outer.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="container mt-4">
+    <p class="alert alert-info">my-outer works!</p>
+    <app-my-inner [outerTotal]="outerTotal" (greaterThan10)="onIncreaseInnerTotal($event)" (smallerThanMinus10)="onDecreaseInnerTotal($event)"></app-my-inner>
```

Comment:
the event emitter should not be named after the number. what if we change it to 5? the function will have a misleading name.